todo_app/flask_config.py

```diff
@@ -1,8 +1,19 @@
 import os
 
-
 class Config:
     """Base configuration variables."""
     SECRET_KEY = os.environ.get('SECRET_KEY')
+    TRELLO_KEY = os.environ.get('TRELLO_KEY')
+    TRELLO_TOKEN = os.environ.get('TRELLO_TOKEN')
+    TRELLO_BOARD = os.environ.get('TRELLO_BOARD')
```

Comment:
Just a heads up for making this work with tests in Module 3:

You are currently setting these variables as class attributes. So they try to read from the environment whenever this file is first imported (i.e. before your tests load the env vars)

You will need to move this setup into an __init__ function and then use Config() instead of Config in app.py. This way, they will be read when you create the app and run `Config()`, which should be after your test has loaded env vars.

Or another approach would be to make them into functions which read from `os.environ` each time you call the function.

Let me know if this is unclear, or ignore me if you have already managed to get your tests working correctly.

Reply:
Hi, wasn't really sure how this was meant to be implemented from your description, but I believe I got round this using the pytest-dotenv plugin.  Are there any problems with this approach?